donghyeun/routes/postRouter.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const postController = require("../controllers/postController");
+
+const router = express.Router();
+
+router.post("/upload", postController.Upload);
+router.get("/all", postController.All);
+router.get("/user/:userId", postController.User);
+router.patch("/modify/:postId", postController.Modify);
+router.delete("/delete/:postId", postController.Delete);
+router.post("/like/:postId", postController.Like);
```

Comment:
endpoint 만들 때 동사는 사용하지 않습니다. 행위는 http method로 나타냅니다.
```suggestion
router.post("", postController.Upload);
router.get("/all", postController.All);
router.get("/user/:userId", postController.User);
router.patch("/:postId", postController.Modify);
router.delete("/:postId", postController.Delete);
router.post("/like/:postId", postController.Like);
```